include/openenclave/internal/plugin.h

```diff
@@ -121,8 +148,8 @@ struct _oe_attester
         size_t* endorsements_buffer_size);
 
     /**
-     * Creates a legacy SGX report to be used in local or remote attestation.
-     * The report shall contain the data given by the **report_data** parameter.
+     * Creates a legacy OE report to be used in SGX local or remote attestation.
+     * The report shall contain the data given by the report_data parameter.
```

Comment:
FYI, technically I think the doxygen-recommended markup for report_data in this line is to prefix it with \p
as discussed at https://stackoverflow.com/questions/15398711/whats-the-right-way-to-reference-a-parameter-in-doxygen

Reply:
Thanks!

"Just a question: why does it need to be wrapped at all?" - good question. But this header is in the existing code, so it is preserved.

The "\p" prefix was not used in existing comments. Is that a style we want to adopt?